index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse week 1</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
+  </body>
+</html>
+https://github.com/Alazar721/hellomicroverse/pulls
+https://github.com/Alazar721/hellomicroverse.git

```

Comment:
- [ Required ] As requested by a previous reviewer please consider removing these links from your `index.html` file since they don't serve any purpose in your project. Currently, the links are displaying on the home page of your project which should not be the case. Please remove them, see the screenshot below for more details on this.

![Screenshot from 2022-12-03 01-44-25](https://user-images.githubusercontent.com/16392046/205402323-b4cc4a99-b1e0-46d1-bb78-397aaec17a6c.png)